platooning_strategic_IHP/src/platoon_manager_ihp.cpp

```diff
@@ -361,13 +361,13 @@ namespace platoon_strategic_ihp
     bool PlatoonManager::insufficientGapWithPredecessor(double distanceToPredVehicle) {
         
         // For normal operation, gap > minGap is necessary. 
-        bool frontGapIsTooSmall = distanceToPredVehicle < minGap_; 
+        bool frontGapIsTooSmall = distanceToPredVehicle < config_.minGap; 
```

Comment:
It feels like a silly question, but I don't see where the config_ member variable is getting populated.  It is declared in the .h file, but there is an empty constructor here, so it seems something is missing.